docs/_static/vcr/v2.yaml

```diff
@@ -0,0 +1,288 @@
+openapi: "3.0.0"
+info:
+  title: Nuts Verifiable Credential API spec
+  description: |
+    API specification for common operations on Verifiable credentials.
+    It allows the three roles, issuer, holer and verifier to issue, revoke, search, present and verify credentials.
+  version: 2.0.0
+  license:
+    name: GPLv3
+servers:
+  - url: http://localhost:1323
+paths:
+  /internal/vcr/v2/issuer/vc:
+    post:
+      summary: Issues a new Verifiable Credential
+      description: |
+        Issues a new Verifiable Credential for provided type in the context.
+        
+        error returns:
+        * 400 - One or more of the given parameters are invalid
+        * 500 - An error occurred while processing the request
+      operationId: "issueVC"
+      tags:
+        - credential
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/IssueVCRequest'
+      responses:
+        "200":
+          description: "New VC has been created successfully. Returns the Verifiable Credential."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/VerifiableCredential'
+        default:
+          $ref: '../common/error_response.yaml'
+
+  /internal/vcr/v2/issuer/vc/search:
+    get:
+      summary: "Searches for verifiable credentials issued by this node which matches the search params"
+      description: >
+        The resolutionResult contains the credential regardless of the validity.
+        The resolutionResult.currentStatus must be checked to see if the credential is valid.
```

Comment:
It now returns a SearchVCResult, not a resolutionResult. It also doesn't contain a currentStatus jsut VC and/or revocation.